compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/AwtWindow.desktop.kt

```diff
@@ -73,19 +74,17 @@ fun <T : Window> AwtWindow(
 
     DisposableEffect(Unit) {
         windowRef.value = create()
-        // We should init a native window even if it is not yet visible.
-        // `pack` method makes window displayable, creates a native window, init graphic
-        // context, performs the first composition and runs effects
-        if (!currentVisible) {
-            window().pack()
-        }
         onDispose {
             dispose(window())
         }
     }
 
     UpdateEffect {
-        update(window())
+        val window = window()
```

Comment:
```
window().apply {
   update(this)
   if (!isDisplayable) {
            makeDisplayable()
   }
}
```
could be a bit more terse.

Reply:
Do you mean `isDisplayable = true`? We can't do that, because we can't make component `isDisplayable = false`

Reply:
Nope, just make window an implicit receiver.

Reply:
O, I understand. I am not fan of `apply`, because it adds new indentation. two indentation is worse than one

Reply:
Matter of taste, I know a lot of people and articles, that argues against let/apply/etc, when it is used everywhere. I am in the middle, but here, I think, better without it.